integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/respondBE.xml

```diff
@@ -0,0 +1,28 @@
+<!--
+  ~ Copyright (c)2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<proxy xmlns="http://ws.apache.org/ns/synapse" name="respondBE"
+       startOnLoad="true"
+       trace="disable" transports="http https">
+    <description/>
+    <target>
+        <inSequence>
+            <log level="full"/>
```

Comment:
Would having a log cutom with a unique string be more useful? Also it would be less bulky.

Reply:
The issue happens when we have log full only when chunking disabled with ESB backend.

Reply:
:+1: